### PR TITLE
Reservation

### PR DESCRIPTION
**Pull request by Kaji-U**
It's READY !

### CODE REVIEW

**Review by shog189 — APPROVED** [review]
:+1:

File: src/AppBundle/Repository/BaptismHasUserRepository.php [hunk 1]
```diff
@@ -121,5 +121,15 @@ public function findByUserAndRole(User $user, $role){
 
         return $results;
     }
+
```
Comment (ngdo-pro) [inline src/AppBundle/Repository/BaptismHasUserRepository.php]: This is a useless function

**Review by wafaworks — APPROVED** [review]
🐰

File: src/UserBundle/Controller/MemberController.php [hunk 2]
```diff
@@ -32,32 +33,31 @@ public function orderAction(Request $request){
 
         $form = $this->createFormBuilder()
             ->add('emails', CollectionType::class, array(
-                    'entry_type'    => EmailType::class,
-                    'allow_add'     => true,
-                    'allow_delete'  => true,
+                    'entry_type' => EmailType::class,
```
Comment (ngdo-pro) [inline src/UserBundle/Controller/MemberController.php:36]: can you redress the arrows please ? For a better lisibility, thx

**Review by ngdo-pro — CHANGES_REQUESTED** [review]

**Review by ngdo-pro — APPROVED** [review]

**Pull request merged**